libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -29,9 +29,10 @@
 #include "config.h"
 
 /* MQTT packet types. */
-#define MQTT_PACKET_TYPE_CONNECT         ( ( uint8_t ) 0x10U ) /**< @brief CONNECT (client-to-server). */
-#define MQTT_PACKET_TYPE_CONNACK         ( ( uint8_t ) 0x20U ) /**< @brief CONNACK (server-to-client). */
-#define MQTT_PACKET_TYPE_DISCONNECT      ( ( uint8_t ) 0xE0U ) /**< @brief DISCONNECT (client-to-server). */
+#define MQTT_PACKET_TYPE_CONNECT         ( ( uint8_t ) 0x10U )
+#define MQTT_PACKET_TYPE_CONNACK         ( ( uint8_t ) 0x20U )
+#define MQTT_PACKET_TYPE_PUBLISH         ( ( uint8_t ) 0x30U )
+#define MQTT_PACKET_TYPE_DISCONNECT      ( ( uint8_t ) 0xE0U )
```

Comment:
This seems to remove the comments. Please revert

Reply:
Good catch thanks